android/app/src/main/java/com/mattermost/rnbeta/MainApplication.java

```diff
@@ -88,6 +104,16 @@ public boolean isDebug() {
             new RNCWebViewPackage(),
             new RNGestureHandlerPackage()
     );
+
+    // Uncomment to load module for build that has telemetry enabled
+    // packages.add(new StartTimePackage(this));
```

Comment:
Are these lines going to be uncommented as part of our build process?

Reply:
Yes, although this particular line is now removed.
It's still the way it is with `// addReactMarkerListener();`